src/main/java/step1/StringUtils.java

```diff
@@ -0,0 +1,14 @@
+package step1;
+
+public class StringUtils {
+
+	public static String[] split(String inputString) {
+		DelimiterExtractor extractor = new DelimiterExtractor();
+		String integers = extractor.separateDelimiterAndIntegers(inputString);
+		return integers.split(extractor.getDelimiter());
+	}
+
+	public static boolean isEmpty(String inputString) {
+		return null == inputString || inputString.isEmpty();
```

Comment:
```suggestion
		return Obejcts.isNull(inputString) || inputString.isEmpty();
```
Objects.isNull()을 찾아보고 활용해보세요 !